addons/mail/static/src/messaging/entity/message/message.js

```diff
@@ -371,6 +385,20 @@ function MessageFactory({ Entity }) {
             );
         }
 
+        /**
+         * @private
+         * @returns {boolean}
+         */
+        _computeIsNeedaction() {
+            const inbox = this.env.entities.Thread.find(
+                thread => thread.model === 'mail.box' && thread.id === 'inbox'
+            );
+            if (!inbox) {
+                return false;
+            }
+            return this.allThreads.includes(inbox);
```

Comment:
Do we need to link this compute to inbox? Don't we have a way to store the needaction information differently?

Reply:
The source of truth was initially designed of message being needaction when linked to inbox, so that's why we have to compute `isNeedaction` from this relation. I think it would make more sense the other way now that we have computed fields, but I wouldn't make this change due to links between `message` and `thread` being on `ThreadCache` and that's tricky, so unsafe in regard to this PR.

Reply:
ok